### PR TITLE
Corrected LTFE location

### PR DESCRIPTION
**Pull request by within7**
Corrected LTFE location

### CODE REVIEW

File: VATSpy.dat [hunk 1]
```diff
@@ -8544,7 +8544,7 @@ LSZU|Buttwil|47.264722|8.3025||LSAS|0
 LSZV|Sitterdorf|47.508889|9.262778||LSAS|0
 LSZW|Thun|46.756389|7.600556||LSAS|0
 LSZX|Schaenis|47.171667|9.039444||LSAS|0
-LTAB|Ankara-Guvercinlik|39.93306|32.75||LTAA|0
```
Comment (Adrian2k) [inline VATSpy.dat:8547]: Any reason for this?
Why is the name now only "-Guvercinlik" with the - symbol?

File: VATSpy.dat [hunk 2]
```diff
@@ -17862,6 +17862,7 @@ LRBB|Bucharest||LRBB
 LSAS|Swiss|LSAG|LSAS
 LSAS|Swiss|LSAZ|LSAS
 LTAA|Ankara|LTBA|LTAA
+LTBB|Istanbul|LTBA|LTBB
```
Comment (Adrian2k) [inline VATSpy.dat:17865]: Any reason why both LTAA and LTBB has LTBA as prefix position?

**Review by NelisV — CHANGES_REQUESTED** [review]
@within7 please have a look at adrian's comments

**Comment by Adrian2k** [comment]
Closing this as there has been no reply.

**Pull request closed without merging**